app.js

```diff
@@ -1,123 +1,75 @@
 'use strict';
 
+var myStores = [];
 var storeHours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '8pm', 'total'];
-//15 hours in day
 
-// create ul tag
-// create one li tag
-// fill li tag innertext w/ information
-// append li tag to ul as child
-// append ul to body
-// var element = document.createElement(tagName[, options]);
+var body = document.getElementsByTagName('body')[0];
 
-
-var firstPike = {
-  name: '1st and Pike',
-  minCust: 23,
-  maxCust: 65,
-  avgCookie: 6.3,
-  hourlysales: [],
-
-  randomCustomerPerHour: function(){
-    return Math.floor(Math.random() * ((this.maxCust - this.minCust) + this.minCust));
-  },
-  sales: function(){
-    var total= 0;
-    for (var i = 0; i <= storeHours.length - 1; i++){
-      this.hourlysales.push(Math.ceil(this.randomCustomerPerHour() * this.avgCookie));
-      console.log('sales is: ' + this.hourlysales);
-      total = total + this.hourlysales[i];
-      console.log('total is: ' + total);
+function Store(storeName, minCust, maxCust, avgCookies){
+  this.name = storeName;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgCookies = avgCookies;
+  this.salesArr = [];
+  this.randCust = function(){
+    return Math.floor(Math.random() * (this.maxCust - this.minCust + 1) + this.minCust);
+    console.log(this.randCust);
```

Comment:
Use console.log for dev work to verify your code does what you think and hunting down bugs.   Once you know a function is working how you expect or a bug is resolved you should remove your console.logs